File: backend/src/main/java/org/jdk/project/dto/digitaltwin/DigitalTwinOverviewDto.java
package org.jdk.project.dto.digitaltwin;

import java.util.List;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DigitalTwinOverviewDto {
  private List<DigitalTwinRobotDto> robotList;
  private List<DigitalTwinPassengerDto> passengerList;
  private List<DigitalTwinInspectionDto> inspectionList;
}

File: backend/src/main/kotlin/org/jdk/project/dto/digitaltwin/DigitalTwinOverviewDto.kt
package org.jdk.project.dto.digitaltwin

data class DigitalTwinOverviewDto(
    var robotList: List<DigitalTwinRobotDto>? = null,
    var passengerList: List<DigitalTwinPassengerDto>? = null,
    var inspectionList: List<DigitalTwinInspectionDto>? = null,
)
